cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -194,6 +194,15 @@ body {
         &-link,
         &-overview-link {
             width: 100%;
+
+            &__current,
+            &:hover {
+                cursor: pointer;
```

Comment:
Is this necessary? 2nd-level links are styled with the `cursor: default` rule on L891 and 1st level links default to `pointer` as far as I can tell.

Reply:
I got these style updates directly from @anselmbradford; if there is a better or easier way to do this, please let me know and I'd be happy to change!

Reply:
Is this still styling a link? I think this was in there because they were links but we styled to look like they weren't links.